### DISCUSSION TITLE
Addon idea: "UI Tweaks"

### DISCUSSION THREAD

**Discussion by Waakul** [ScratchAddons/ScratchAddons#7254] [Ideas]
### Pull Request For Addon: 
**No, I wont create a pull request until at least two maintainers or a lead maintainer approves this idea and there are enough pull requests of settings to add**

Sometimes I think that scratch's UI is a bit boring so this addon would be helpful with settings that makes tweaks in the UI.

### Suggestion:
An addon named "Website UI Tweaks" containing lots of Website UI tweaks which are toggleable because others are creating UI tweaks as separate addons so we could make this a setting in 1.

### Ideas:
**Ideas for settings:**
- [ ] Blur Website Background On Popups: None
- [ ] 3.0 styled containers: None
- [ ] Rounded Corners On Cards, Profile pictures etc: None
Suggest Ideas Below In The Following Format: 
```
Setting Name: <Name>
Setting Description: <Desc>
Pull Request: <Pull Request> (If it exists)
````
**Addons that already exist that could be added:**

THIS LIST IS INCOMPLETE!, Let me know other addons that could be added as a reply to this.
Queries must be replies to this to keep the discussion clean.

**Comment by Secret-chest** [comment]
Would make it harder to maintain.

**Reply by Waakul** [reply]
pls write replies as a reply instead of an answer (or comment)

**Comment by CST1229** [comment]
Website and editor addons are separated (see: how editor dark mode and website dark mode are different addons, and customizable navigation bar vs. customizable menu bar), so the options for the website and the editor should be in different addons.

**Reply by Waakul** [reply]
hmm ok

**Reply by Waakul** [reply]
Please write replies as a reply instead of an answer (or comment)

**Comment by Secret-chest** [comment]
Separate addons make it clearer, are much easier to maintain, allow featuring only some tweaks.

Also, this would create an inconsistency since merging previous addons into this would require hard migrations.

**Reply by Waakul** [reply]
only minor tweaks will be added ig

**Comment by jazzathedev** [comment]
Did we not make an issue for the 2.0 tweaks idea? This should be closed in favour of that imo.

**Reply by Waakul** [reply]
where?

**Comment by Waakul** [comment]
Low support for the idea

**Reply by Waakul** [reply]
closed
